Kitodo/src/main/java/de/sub/goobi/config/ConfigCore.java

```diff
@@ -33,15 +33,7 @@
     private static ServiceManager serviceManager = new ServiceManager();
     private static final String METADATA_DIRECTORY = "MetadatenVerzeichnis";
     private static final String CONFIG_DIR = "KonfigurationVerzeichnis";
-
-    /**
-     * Get the path for the temporary images directory.
-     *
-     * @return the path for the temporary images directory as String
-     */
-    public static String getTempImagesPath() {
-        return "/pages/imagesTemp/";
-    }
+    public static String imagesTemp = "/pages/imagesTemp/";
```

Comment:
Why are you replacing this? Class member variables should never accessed directly as a public member variable. Or should this be a constant value?

Reply:
Constant.

Reply:
Then please change it to a constant according our coding and checkstyle rules.